components/DesktopInventoryItemList/index.tsx

```diff
@@ -9,19 +9,9 @@ import TablePagination from '@mui/material/TablePagination'
 import TableRow from '@mui/material/TableRow'
 import TableSortLabel from '@mui/material/TableSortLabel'
 import { visuallyHidden } from '@mui/utils'
-import DesktopInventoryItemListItem from 'components/DesktopInventoryItemList/DesktopInventoryItemListItem'
+import InventoryItemListItem from 'components/DesktopInventoryItemList/DesktopInventoryItemListItem'
```

Comment:
```suggestion
import InventoryItemListItem from 'components/DesktopInventoryItemList/InventoryItemListItem'
```
Rename the file that this component comes from to `InventoryItemListItem.tsx` and update relavant imports

Reply:
Looked at this again and now I'm confused. The component name in `DesktopInventoryItemListItem.tsx` is `DesktopInventoryItemListItem` but it is being imported as `InventoryItemListItem`...